fern/apis/generators-yml/definition/generators.yml

```diff
@@ -337,9 +340,9 @@ types:
       overrides: optional<string>
       namespace: optional<string>
       settings: optional<OpenAPISettingsSchema>
-  
-  OpenRPCSpecSchema: 
-    properties:   
+
```

Comment:
ok I can see how this diff is annoying, but I want to lint the yaml files so bad haha :)